resources/public/api/conf/1.0/common/headers.yaml

```diff
@@ -0,0 +1,53 @@
+components:
+  parameters:
+    acceptJson:
+      name: "Accept"
+      in: header
+      description: |
+        Specifies the response format and the version of the API to be used.
+      schema:
+        type: string
+        enum:
+          - "application/vnd.hmrc.1.0+json"
+      required: true
+
+    authorization:
```

Comment:
Could we replace this with "authorizationWrite" and "authorizationRead" or something like [this](https://github.com/hmrc/property-business-api/blob/c8d4daf151e516da59371527d87f1cf645a0f4c5/resources/public/api/conf/1.0/common/headers.yaml#L14)?

Reply:
Done!